code_search/kubeflow/components/submit-code-embeddings-job.jsonnet

```diff
@@ -8,6 +8,7 @@ local baseParams = std.extVar("__ksonnet/params").components["submit-code-embedd
 local experimentName = baseParams.experiment;
 local params = baseParams + experiments[experimentName] + {
   name: experimentName + "-embed-code",
+  bqSuffix: std.strReplace(self.jobNameSuffix, "-", "_"),
```

Comment:
See comment below. I think this should probably be done in experiments.libsonnet. For each experiment we should define the complete path/name of the things we care about (e.g. the individual BQTables). This makes the code more readable; i.e. they can look at experiments.libsonnet and see the full path of the BQ table for the function embeddings table.

Reply:
I didn't put the parameter for 'pipeline' env in experiments because it's not able to be overriden. I move them in param.libsonnet instead.